index.html

```diff
@@ -34,15 +34,7 @@ <h1>I’m Tshepo. <br> Glad to see <br> you!</h1>
   
 </div>
 
-
-
-
-
-
-   
-
-
-    <div class="Container2">
+   <div class="Container2">
       <img class="Snapshoot2" src="image/SnapshootD3.png" alt="">
       <h2 class="Tonic">  Tonic</h2>
```

Comment:
- [ ] Your project title seems to have different sizes, some are small and some are bigger, I'll suggest you go to the Figma design and get the correct size and implement it across all the sizes. :+1: 

- -  This is small:
![Screenshot from 2023-06-02 12-25-13](https://github.com/Tshepo45/Mobile-Finish/assets/107632780/c2224c69-3b7d-43d4-8bf0-cce42d1ed854)

- - And this is bigger:
![Screenshot from 2023-06-02 12-24-56](https://github.com/Tshepo45/Mobile-Finish/assets/107632780/d25ad45b-d6ef-4a60-836c-35f6fba82ca8)